wkhtmltopdf/utils.py

```diff
@@ -231,14 +235,22 @@ def make_absolute_paths(content):
 
     return content
 
-def render_to_temporary_file(template, context, mode='w+b', bufsize=-1,
-                                 suffix='.html', prefix='tmp', dir=None,
-                                 delete=True):
-    # make sure the context is a context object
-    if not isinstance(context, (Context, RequestContext)):
-        context = Context(context)
+def render_to_temporary_file(template, context, request=None, mode='w+b',
+                             bufsize=-1, suffix='.html', prefix='tmp',
+                             dir=None, delete=True):
+    if django.VERSION < (1, 8):
+        # If using a version of Django prior to 1.8, ensure ``context`` is an
+        # instance of ``Context``
+        if not isinstance(context, Context):
+            if request:
+                context = RequestContext(request, context)
+            else:
+                context = Context(context)
+        content = template.render(context)
+    else:
+        content = template.render(context, request)
```

Comment:
I'm a bit confused about the `render` method's signature. The doc seems to be stating something different than what you did here: https://docs.djangoproject.com/en/1.9/ref/templates/api/#django.template, and so does the code https://github.com/django/django/blob/master/django/template/base.py#L200. Am I looking at the wrong method? Sorry if that's something obvious.